README.md

```diff
@@ -574,19 +574,93 @@ Returns the preferred_terminology at the Terminology("self") level. <br> Expecte
 }
 ```
 
-#### POST
+#### PUT
 
-Create a reference to a prefered `Terminology` for the terminology (specified by id). <br> 
+Create or replace references to a prefered `Terminology` for the terminology (specified by id). <br> 
 Request body example:
  ```json
-[
-    {
-        "preferred_terminology": "tm--example3"
-    },
-    {
-        "preferred_terminology": "tm--example6"
-    }
-]
+{
+    "editor": "me",
+    "preferred_terminologies": [
+        {
+            "preferred_terminology": "tm--example1"
+        },
+        {
+            "preferred_terminology": "tm--example6"
+        }
+    ]
+}
+```
+
+#### DELETE
+Running the DELETE request will remove the preferred_terminology collection
+from the `Terminology` specified by the id
+
+### https://[APPURL]/api/Terminology/[id]/user_input/[code]/[type]
+  
+  * id (str): The document ID.
+  * code (str): The target document (mapping) identifier.
+  * type (str): The type of input to retrieve 
+     - currently available: "mapping_conversations","mapping_votes".
+
+#### GET
+
+Return the reference to the `user_input` `type` related to the `Terminology` (specified by id).<br>
+Returns all records of the `type` at the `code` level. <br> 
+
+Expected return for type `mapping_votes` below
+
+```json
+{
+    "Terminology": "tm--2VjOxekLP8m28EPRqk95",
+    "code": "TEST_0001",
+    "mapping_votes": [
+        {
+            "date": "Oct 04, 2024, 03:35:23.088717 PM",
+            "user_id": "test_session",
+            "vote": "up"
+        }
+    ]
+}
+```
+
+Expected return for type `mapping_conversations` below
+
+```json
+{
+    "Terminology": "tm--2VjOxekLP8m28EPRqk95",
+    "code": "TEST_0001",
+    "mapping_conversations": [
+        {
+            "date": "Oct 04, 2024, 04:17:43.043579 PM",
+            "user_id": "test_session",
+            "note": "I like this mapping"
+        },
+        {
+            "date": "Oct 04, 2024, 04:21:15.460040 PM",
+            "user_id": "test_session2",
+            "note": "I dont like this mapping"
+        }
+    ]
+}
+```
```

Comment:
For simplicity/ease of use I chose to give the same format to mapping_votes as for mapping_conversations. I don't think that was our final thought at the end of our last conversation. Let me know what you think.

Reply:
I think my only concern there is that we only want one vote per person, so if they are in a list and the user changes their vote, the backend will need to iterate over and remove or overwrite the vote with their current choice.